schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals(
+   id INT,
```

Comment:
[OPTIONAL] This requirement is optional but recommended so kindly don't skill this requirement as mentioned by the previous :point_up:  reviewer it would be nice if we mark the id column as the primary key, not null and autoincremented so kindly add this optional requirement :slightly_smiling_face: